pixano/datasets/dataset.py

```diff
@@ -688,6 +691,111 @@ def get_embedding(
         """
         return self.get_embedding(idx, 1, select)[0]
 
+    def save_item(
```

Comment:
I do not think that save_item should handle Front objects. IMO the front should only communicate with the REST API which should handle the conversion for the back.

The back should only handle lance models so that DS can provide in a pythonic way new items to save.

Reply:
Right, I moved this function to REST api